code/modules/awaymissions/capture_the_flag.dm

```diff
@@ -138,12 +138,14 @@
 	var/ctf_enabled = FALSE
 	var/ctf_gear = /datum/outfit/ctf
 	var/instagib_gear = /datum/outfit/ctf/instagib
-	var/list/dead_barricades = list()
+
+	var/list/obj/effect/ctf/dead_barricade/dead_barricades = list()
+	var/list/obj/structure/barricade/security/ctf/living_barricades = list()
 
 	var/static/ctf_object_typecache
 	var/static/arena_cleared = FALSE
 
-/obj/machinery/capture_the_flag/New()
```

Comment:
This is the only one I'm worried about. See #24211. But as long as you've tested it more power to you